exercises/concept/annalyns-infiltration/.docs/instructions.md

```diff
@@ -18,7 +18,7 @@ You have four tasks: to implement the logic for determining if the above actions
 Define the `CanFastAttack()` function that takes a boolean value that indicates if the knight is awake. This function returns `true` if a fast attack can be made based on the state of the knight. Otherwise, returns `false`:
 
 ```go
-const knightIsAwake = true;
+var knightIsAwake = true
```

Comment:
I feel like the := syntax is more commonly used for variable declarations. Should we steer students towards that?

Reply:
On the other hand, using the full var declaration is more "context agnostic" as it can be used outside any function. I'm just thinking about the use case where a student might want to copy/paste this and might copy/paste the code to be outside any function - the full var declaration will work.

But I don't really have a strong opinion about this, so if you feel the change to `:=` is better, I'll happily make it.

Reply:
No strong preference on my part, especially if there is a reasonable pedagogical reason for using this one.